RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,55 @@
 
 import Foundation
 
-print("Hello, World!")
+let comNum = Int.random(in: 1..<4)
+var boolGame: Bool = true
 
+
+while(boolGame) {
+    print("컴퓨터의 선택번호 \(comNum)")
+    print("가위(1), 바위(2), 보(3)! <종료: 0> : ")
+
+    guard let userNum = Int(readLine() ?? "") else { throw NSError() }
+    boolGame = RockPaperScissors(user: userNum, com: comNum)
+}
+print("게임 종료")
+
+func RockPaperScissors(user: Int, com: Int) -> Bool{
```

Comment:
```suggestion 
func RockPaperScissors(user: Int, com: Int) -> Bool {
```

중괄호 이전과 다음에는 공백이 들어가는게 좋아요. https://www.swift.org/documentation/api-design-guidelines/ 스위프트 코딩 스타일을 한번 살펴보시는걸 권장드려요.

Reply:
공백 반영했습니다.